docs/build/develop-services.md

```diff
@@ -299,4 +299,157 @@ result: "Hello, fluence"
 3> 
 ```
 ## Mounted binaries
-To unlock some features not provided for wasm, exists a mounted binaries API - a way to call programs on host machine from wasm services.  TODO
\ No newline at end of file
+To unlock some options not possuble in pure wasm, exists a mounted binaries API - a way to call programs on host machine from wasm services. It is similar to the importing functions, the only differences are that `wasm_import_module` must be `"host"` and signatures are predefined.
+
+Lets use the `curl` tool from the host system in our project. First, add import the binary and use it:
```

Comment:
Let's use `curl` from the host system in our project. First, import the binary and use it: